### PR TITLE
Add missing information to amw insights workbooks

### PR DESCRIPTION
**Pull request by msft-alwag**
- Adds limits columns to at-scale view
image

- Adds "unused metrics" section to resource view
image

### CODE REVIEW

File: Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook [hunk 1]
```diff
@@ -293,7 +334,96 @@
...
+			  "version": "KqlParameterItem/1.0",
+			  "parameters": [
+				{
+				  "id": "05607eff-fcbc-40b8-8eda-e36c4b90a6f7",
+				  "version": "KqlParameterItem/1.0",
+				  "name": "notUsedInDays",
+				  "label": "Not Used In Days",
+				  "type": 2,
+				  "isRequired": true,
+				  "query": "{\"version\":\"1.0.0\",\"content\":\"[\\\"0d\\\", \\\"30d\\\", \\\"60d\\\", \\\"90d\\\"]\",\"transformers\":null}",
```
Comment (mrcaps) [inline Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook:369]: Sorry the `0d` option was just for testing - given that Praval's AMW looks to have reasonable data I think we can remove this even at this point so we don't forget about this later (or if needed for testing, if you could create a tracking task)

File: Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook [hunk 2]
```diff
@@ -293,7 +334,96 @@
...
+			  "style": "pills",
+			  "queryType": 8
+			},
+			"name": "parameters - 2"
+		  },
+		  {
+			"type": 3,
+			"content": {
+			  "version": "KqlItem/1.0",
+			  "query": "let daysLimit = tolong(substring(\"{notUsedInDays}\", 0, strlen(\"{notUsedInDays}\") - 1));\nAMWMetricsUsageDetails\n| where DaysSinceMetricQueried >= daysLimit\n| summarize [\"Event Count\"] = avg(IncomingEventsCount), [\"Daily Timeseries\"] = avg(DailyTimeseriesCount), [\"Last Queried (days ago)\"] = min(DaysSinceMetricQueried) by [\"Azure Monitor Workspace\"] = _ResourceId, MetricNamespace, MetricName, DimensionsList\n| summarize [\"Event Count\"] = avg([\"Event Count\"]), [\"Daily Timeseries\"] = sum([\"Daily Timeseries\"]), [\"Last Queried (days ago)\"] = min([\"Last Queried (days ago)\"]) by [\"Azure Monitor Workspace\"], MetricNamespace, MetricName",
```
Comment (mrcaps) [inline Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook:390]: fwiw, the substring thing was a bit of a hack to not have to have separate values/labels - IMO nonblocking and your call on if / whether to switch over at this point, but figured I'd mention

**Review by mrcaps — APPROVED** [review]

File: Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook (hunk 2 above)
Comment (msft-alwag) [reply Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook:390]: I took a quick look at it and got the dropdown kind of working without the hack, but it still worked better with the hack so punting this for now

Comment (mrcaps) [reply Workbooks/Azure Monitor - Workspaces/Usage Insights/Usage Insights.workbook:390]: Sounds good!

**Review by mrcaps — APPROVED** [review]

**Pull request merged**